src/Microsoft.Azure.ServiceBus/ServiceBusConnectionStringBuilder.cs

```diff
@@ -14,12 +15,20 @@ public class ServiceBusConnectionStringBuilder
         const char KeyValueSeparator = '=';
         const char KeyValuePairDelimiter = ';';
         static readonly string EndpointScheme = "amqps";
-        static readonly string EndpointFormat = EndpointScheme + "://{0}.servicebus.windows.net";
         static readonly string EndpointConfigName = "Endpoint";
         static readonly string SharedAccessKeyNameConfigName = "SharedAccessKeyName";
         static readonly string SharedAccessKeyConfigName = "SharedAccessKey";
         static readonly string EntityPathConfigName = "EntityPath";
 
+        string entityPath, sasKeyName, sasKey, endpoint;
+
+        /// <summary>
+        /// Instantiates a new <see cref="ServiceBusConnectionStringBuilder"/>
+        /// </summary>
+        public ServiceBusConnectionStringBuilder()
```

Comment:
>public [](start = 8, length = 6)

If not using this now, may be make this internal, we could make this public later, when we use it right?